pkg/sources/openshift.com/mirror/mirror.go

```
@@ -3,12 +3,13 @@ mirror provides the capability for tools to retrieve files from mirror.openshift
 */
 package mirror
 
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)